Sources/SwiftTypeAdoptionReporter/FastStrategy.swift

```diff
@@ -42,6 +42,18 @@ public class FastStrategy: Strategy {
                 usageCount: usageCounts[type] ?? 0
             )
         }
+
+        if verbose {
+            for type in types {
+                guard let files = fileCounts[type] else { continue }
+
+                print("\(type) used in the following files:")
+                files
+                    .map({  " \($0.pathString)" })
+                    .forEach { print($0) }
```

Comment:
Oh I meant to just do the print inside the forEach, as the map is an O(n) operation as it is, but either way 👍 

```
files.forEach {
   print(" \($0.pathString)")
}
```

Reply:
Oh gotcha, I think it's a bit clearer when separating out each logic step in the transformation, so going to leave as is